.pre-commit-config.yaml

```diff
@@ -17,25 +17,13 @@ repos:
       hooks:
           - id: yamlfmt
 
-    - repo: https://github.com/mgedmin/check-manifest
```

Comment:
This is apparently running `pip install` to build the package so it's quite slow. I think our MANIFEST file is stable enough now.

Reply:
What about removing this in another PR together with switching to flit? IMO, flit does not require a MANIFEST file at all.

Reply:
Sure. I brought it back, but put it at the end because it's the slowest step. (I'll note that my worry is not so much about CI time, and more about local dev experience).